webpack/scenes/ContentViews/Details/Filters/Add/__tests__/cvFilterAdd.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { renderWithRedux, patientlyWaitFor, fireEvent } from 'react-testing-lib-wrapper';
+
+
+import api from '../../../../../../services/api';
+import CVFilterAddModal from '../CVFilterAddModal';
+import { nockInstance, assertNockRequest } from '../../../../../../test-utils/nockWrapper';
+
+const cvCreateData = require('./cvFilterCreateResult.fixtures.json');
+
+const cvCreateFilterPath = api.getApiUrl('/content_view_filters?content_view_id=5');
+
+const setIsOpen = jest.fn();
+
+const createDetails = {
+  name: 'test',
+  description: 'Creating filter',
+  inclusion: true,
+  type: 'rpm',
+};
+
+const createdCVDetails = { ...cvCreateData };
+
+const form = <CVFilterAddModal cvId={5} show setIsOpen={setIsOpen} />;
+
+test('Can save content view filter from form', async (done) => {
```

Comment:
This one doesn't use `await` as far as I can see
```suggestion
test('Can save content view filter from form', (done) => {
```